ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -0,0 +1,48 @@
+# SONiC
+
+# Setup Platform
+#- include: platform-cavm.yml
+
+# Install Cavium Host Interface Driver
+- name: Install driver package
+  become: true
+  apt: pkg=xp80PcieEndpoint
+       state=present
+       force=yes
+
+# Setup port SKU
+- name: Setup port configuration file
+  become: yes
+  template: src=etc/ssw/{{ sonic_hwsku }}/profile.ini.j2
```

Comment:
Missing this template file in the commit? Is the content static or dynamic? Can we build it into syncd_cavm docker image, so we can keep the syncd_docker_volumes_dict as simple as possible, also platform consistent.